go.mod

```diff
@@ -7,3 +7,5 @@ require (
 	github.com/terraform-providers/terraform-provider-tls v1.2.0
 	golang.org/x/oauth2 v0.0.0-20190604054615-0f29369cfe45
 )
+
+go 1.13
```

Comment:
It may be safer to leave this particular change for a separate PR.

We are still in the stage of understanding all implications of go 1.13 on the provider ecosystem, e.g. its enforcement of certain format of import paths and way of versioning.

That is not to say that we can't upgrade to 1.13, but I would suggest caution in that regards and perhaps cut a separate provider (patch or minor) release with just Go upgrade, so we can isolate potential issues.